app/models/card.rb

```diff
@@ -8,9 +8,8 @@ class Card < ActiveRecord::Base
   after_initialize :set_status, if: :new_record?
 
   enum card_type: { flashcard:     0,
-                    short_answer:  1,
-                    organize:      2,
-                    fill_in_blank: 3 }
+                    organize:      1,
+                    fill_in_blank: 2 }
```

Comment:
so once an enum is created, it should generally maintain it's integer value _forever_. Changing the order is fine, but the integer should stay the same. defining the integer at all is completely optional. if an int is not defined, it is assigned based on the order, which creates problems if they are reordered and the ints are not defined, that is why i defined the integers.

however, in this case, because we have no production data yet, this will have no negative impact. I just want you to be aware that enums are a little finicky that way